docs/reference/cluster-config.txt

```diff
@@ -352,6 +352,8 @@ The master supports the following configuration settings:
 -  ``root``: Specifies the root directory of the state files. Defaults
    to ``/usr/share/determined/master``.
 
+-  ``cluster_name``: Specify a human readable name for this cluster.
```

Comment:
We should indicate if this is a required or optional field. (I am assuming optional otherwise would be a breaking change).

Reply:
+1